Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2823,8 +2828,31 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin(const URL& url)
         }), this);
 }
 
+void MediaPlayerPrivateGStreamer::configureDepayloader(GstElement* depayloader)
+{
+#if ENABLE(MEDIA_STREAM)
+    if (!WEBKIT_IS_MEDIA_STREAM_SRC(m_source.get()))
+        return;
+
+    auto depayloaderHasProperty = [&depayloader](const char* name) -> bool {
+        return g_object_class_find_property(G_OBJECT_GET_CLASS(depayloader), name);
+    };
```

Comment:
I prefer to do this kind of functions as statics, but your call.

Reply:
For a one-liner like this I suppose both approaches work :)